scripts/data/fullhog/d3-android-fullhog.txt

```diff
@@ -0,0 +1,112 @@
+AIGAMEi.STR
```

Comment:
I guess this file should not have been committed?

Reply:
this i just copied from `d3-linux-fullhog.txt` actually. currently it just satisfies CMake's target-graph. haven't actually tested any of it.